k8s/backend-secrets.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backend-secrets
+  namespace: chat-app
+type: Opaque
+stringData:
+  jwt-secret: "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJPbmxpbmUgSldUIEJ1aWxkZXIiLCJpYXQiOjE3MzQ3MjA5MzAsImV4cCI6MTc2NjI1NjkzMCwiYXVkIjoidHJhaW53aXRoc2h1YmhhbS5jb20iLCJzdWIiOiJqcm9ja2V0QGV4YW1wbGUuY29tIiwiR2l2ZW5OYW1lIjoiQWZ6YWwiLCJTdXJuYW1lIjoiUm9ja2V0IiwiRW1haWwiOiJqcm9ja2V0QGV4YW1wbGUuY29tIiwiUm9sZSI6WyJEZXZPcHMiLCJQcm9qZWN0IEFkbWluaXN0cmF0b3IiXX0.ehelBbUU2IFJ8M5xzQL_UFatMxCMSojAOqWQaZQgrwk"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)